doc/whats-new.rst

```diff
@@ -25,6 +25,9 @@ Breaking changes
 
 New Features
 ~~~~~~~~~~~~
+- :py:func:`xarray.concat` now preserves attributes from the first Variable.
```

Comment:
What did it do before?

Reply:
It only kept attributes if they were consistent across all variables. The equality check was also the source of buggy behaviour #2060, #2575